Px.Utils/Validation/ContentValidation/ContentValidationFunctions.cs

```diff
@@ -0,0 +1,973 @@
+﻿using PxUtils.PxFile;
+using PxUtils.Validation.SyntaxValidation;
+
+namespace PxUtils.Validation.ContentValidation
+{
+    public delegate ValidationFeedbackItem[]? ContentValidationEntryFunctionDelegate(ValidationStructuredEntry entry, PxFileSyntaxConf syntaxConf, ref ContentValidationInfo info);
+    public delegate ValidationFeedbackItem[]? ContentValidationSearchFunctionDelegate(ValidationStructuredEntry[] entries, PxFileSyntaxConf syntaxConf, ref ContentValidationInfo info); 
```

Comment:
Maybe "RequiredKeywordValidationDelegate" would be more descriptive?

Reply:
Not all of the entries that are searched by these functions are required necessarily. I find ContentValidationFindKeywordDelegate more descriptive